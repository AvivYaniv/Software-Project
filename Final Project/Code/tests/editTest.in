edit
print_board
edit pathToFileThatDoesntExist.txt
edit tests/b2_2.txt
print_board
undo
redo
reset
print_board
set 4 4 4
set 1 4 1
set 2 4 1
mark_errors 0
print_board
save tests/observed/edit_out_board1.txt
set 1 4 2
save tests/observed/edit_out_board2.txt
edit tests/b2_2_empty.txt
print_board
undo
exit