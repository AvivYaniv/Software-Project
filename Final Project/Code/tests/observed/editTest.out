Sudoku
------
Enter your command:
----------------------------------------
|            |            |            |
|            |            |            |
|            |            |            |
----------------------------------------
|            |            |            |
|            |            |            |
|            |            |            |
----------------------------------------
|            |            |            |
|            |            |            |
|            |            |            |
----------------------------------------
Enter your command:
----------------------------------------
|            |            |            |
|            |            |            |
|            |            |            |
----------------------------------------
|            |            |            |
|            |            |            |
|            |            |            |
----------------------------------------
|            |            |            |
|            |            |            |
|            |            |            |
----------------------------------------
Enter your command:
Error: File cannot be opened
Enter your command:
-------------------
|  1   2 |  4     |
|        |      1 |
-------------------
|        |      2 |
|        |        |
-------------------
Enter your command:
-------------------
|  1   2 |  4     |
|        |      1 |
-------------------
|        |      2 |
|        |        |
-------------------
Enter your command:
Error: no moves to undo
Enter your command:
Error: no moves to redo
Enter your command:
Board reset
Enter your command:
-------------------
|  1   2 |  4     |
|        |      1 |
-------------------
|        |      2 |
|        |        |
-------------------
Enter your command:
-------------------
|  1   2 |  4     |
|        |      1 |
-------------------
|        |      2 |
|        |      4 |
-------------------
Enter your command:
-------------------
|  1*  2 |  4     |
|        |      1 |
-------------------
|        |      2 |
|  1*    |      4 |
-------------------
Enter your command:
-------------------
|  1*  2 |  4     |
|        |      1 |
-------------------
|        |      2 |
|  1*  1*|      4 |
-------------------
Enter your command:
ERROR: invalid command
Enter your command:
-------------------
|  1*  2 |  4     |
|        |      1 |
-------------------
|        |      2 |
|  1*  1*|      4 |
-------------------
Enter your command:
Error: board contains erroneous values
Enter your command:
-------------------
|  1   2 |  4     |
|        |      1 |
-------------------
|        |      2 |
|  2   1 |      4 |
-------------------
Enter your command:
Saved to: tests/observed/edit_out_board2.txt
Enter your command:
-------------------
|        |        |
|        |        |
-------------------
|        |        |
|        |        |
-------------------
Enter your command:
-------------------
|        |        |
|        |        |
-------------------
|        |        |
|        |        |
-------------------
Enter your command:
Error: no moves to undo
Enter your command:
Exiting...
